OpenAI-Extension.sln

```diff
@@ -22,8 +22,6 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "samples", "samples", "{865C
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "chat", "chat", "{46888A9D-8058-499C-9F2F-07800FF686EB}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "ChatBotSample", "samples\chat\csharp-inproc\ChatBotSample.csproj", "{2C15A452-14A2-4156-95E2-E6D709F383EF}"
```

Comment:
the samples are still existing in the repo, just removed from the solution, need to delete that folder manually and any references in the Readme should be updated too

Reply:
Removed it and updated read me!